assets/styles/vendor/_chroma-styles.scss

```diff
@@ -2,7 +2,7 @@
 /* Error */ .chroma .err { color: #000000 }
 /* LineTableTD */ .chroma .lntd { vertical-align: top; padding: 0; margin: 0; border: 0; }
 /* LineTable */ .chroma .lntable { border-spacing: 0; padding: 0; margin: 0; border: 0; width: auto; overflow: auto; display: block; }
-/* LineHighlight */ .chroma .hl { display: block; width: 100%;background-color: #ffffcc }
+/* LineHighlight */ .chroma .hl { display: block; width: 100%; border-left: 4px solid #632ca6; background-color: #c987ff1d; }
```

Comment:
Looks like the border is bumping the text of the code in a little, which is probably not desirable, since incorrect indents are a peeve for some engineers. But you could probably use a box shadow instead, like [this](https://stackoverflow.com/questions/29404093/css-outline-left-and-right)?

Reply:
@jhgilbert  Good eye! I didn't even notice that. I replaced this with box-shadow, and that seems to have resolved it. Thanks!